examples/cockroachdb/README.md

```diff
@@ -0,0 +1,269 @@
+# CockroachDB
+
+[CockroachDB](https://www.cockroachlabs.com/) is one of the database servers based on SQL.
+
+## Introduction
+
+This chart bootstraps a three node CockroachDB deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.16+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+- Kanister controller version 0.80.0 installed in your cluster, let's assume in Namespace `kanister`
+- Kanctl CLI installed (https://docs.kanister.io/tooling.html#install-the-tools)
+
+## Installing the Chart
+
+The commands mentioned below could be run to install the CockroachDB database using the `cockroachdb` chart with the release name `cockroachdb-release`:
+
+```bash
+# Add cockroachdb in your local chart repository
+$ helm repo add cockroachdb https://charts.cockroachdb.com/
+
+# Update your local chart repository
+$ helm repo update
+
+# Install the cockroachdb in cockroachdb namespace
+$ helm install cockroachdb-release --namespace cockroachdb --create-namespace cockroachdb/cockroachdb
+
+# Install the secure cockroachdb client
+# Download latest secure client manifest 
```

Comment:
we're using cockroachdb secure client, so that user could interact with the cockroachdb cluster. Initially user would have to do some activity like insert data into tables etc. so for that user have to create a secure client pod, from the blueprint perspective, we're dynamically creating the client pods using KubeTask